sentry_sdk/integrations/aws_lambda.py

```diff
@@ -214,3 +224,32 @@ def _get_url(event, context):
     if proto and host and path:
         return "{}://{}{}".format(proto, host, path)
     return "awslambda:///{}".format(context.function_name)
+
+
+def _get_cloudwatch_logs_url(context, start_time):
+    # type: (LambdaContext, datetime) -> str
+    """
+    Generates a CloudWatchLogs console URL based on the LambdaContext object
+
+    Arguments:
+        context {LambdaContext} -- context from lambda handler
+
+    Returns:
+        str -- AWS Console URL to logs.
+    """
+    start_time_str = (start_time - timedelta(seconds=1)).isoformat().split(".")[0]
+    end_time_str = (datetime.now() + timedelta(seconds=2)).isoformat().split(".")[0]
+    url = ''.join([
```

Comment:
Instead of this please do

```
url = (
    "string 1"
    "string 2"
)
```

Reply:
That syntax will only work with quoted strings, half of these are variables/functions. I can do a multiline string with `.format` at the end, is that what you want?

Reply:
(fixed)

Reply:
yeah sorry I didn't even think about that. `.format` is good